src/App.tsx

```diff
@@ -15,36 +17,103 @@ export const goodsFromServer = [
   'Garlic',
 ];
 
+enum SortField {
+  name = 'name',
+  length = 'length',
+}
+
+interface Filter {
+  field: SortField;
+  isreverse: boolean;
+}
+
+const preparedGoods = (
+  goods: string[],
+  { field, isreverse }: Filter,
```

Comment:
```suggestion
  { sortType, isReversed }: Filter,
```
What `field` is it? Describe your variables/arguments better and they should be in camelCase